lib/Mojolicious/Plugin/CGI.pm

```diff
@@ -203,8 +204,7 @@ sub register {
       defined($pid = fork) or die "[CGI] Could not fork $name: $!";
 
       unless ($pid) {
-
-        # No need to Mojo::IOLoop->reset, since the CGI script is started with exec()
+	Mojo::IOLoop->reset if $self->{run}; # clean up unless CGI script is started with exec()
```

Comment:
Sure, I'll try. I didn't want to move it because I don' quite understand how it will interact with the open/select statements.